.travis/run.sh

```diff
@@ -123,7 +123,6 @@ for BOARD in $BOARDS; do
 			# Not currently built so use .bit instead
 			#cp ../third_party/misoc/build/*.xsvf $COPY_DEST
 			cp ../third_party/misoc/build/*.bit $COPY_DEST
-			cp ../build/output.*.log $COPY_DEST
```

Comment:
The problem is not knowing what the log is called. I guess there would be no reason not to just always call it output.log instead of what travis called it?